sequencer/src/api.rs

```diff
@@ -524,11 +528,31 @@ pub mod test_helpers {
             bind_version: V,
         ) -> Self {
             let mut cfg = cfg;
-            let (builder_task, builder_url) =
-                run_test_builder::<{ NUM_NODES }>(cfg.network_config.builder_port()).await;
+            let mut builder_tasks = Vec::new();
+            let mut legacy_builder_url = "http://example.com".parse().unwrap();
+            let mut marketplace_builder_url = "http://example.com".parse().unwrap();
+
+            if <V as Versions>::Base::VERSION <= MarketplaceVersion::VERSION {
+                let (task, url) =
+                    run_test_builder::<{ NUM_NODES }>(cfg.network_config.builder_port()).await;
+                builder_tasks.push(task);
+                legacy_builder_url = url;
+            };
+
+            if <V as Versions>::Upgrade::VERSION >= MarketplaceVersion::VERSION {
+                println!("Running marketplace builder!");
+                let (task, url) = run_marketplace_builder::<{ NUM_NODES }>(
+                    cfg.network_config.builder_port(),
```

Comment:
we can run both builders simultaneously so different ports?

Reply:
Yes, for example in case we upgrade from 0.2 -> 0.3

Reply:
Oh, I misinterpreted your comment. Fixed!